internal/cmd/plugin/command_install_test.go

```diff
@@ -0,0 +1,55 @@
+package plugin
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/hashicorp/go-version"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+func TestGetPluginManifest(t *testing.T) {
+	dir, _ := filepath.Abs("../../../test/fixtures/input/plugin")
+	manifest, err := getPluginManifest("confluent-test_plugin", dir)
+	assert.NoError(t, err)
+
+	referenceManifest := &Manifest{
+		Name:        "confluent-test_plugin",
+		Description: "Does nothing",
+		Dependencies: []Dependency{
+			{
+				Dependency: "Python",
```

Comment:
Dependency is already the name of the struct, so this may avoid some confusion. `Dependencies[0].Dependency` is less readable than `Dependencies[0].Name`
```suggestion
				Name: "Python",
```

Reply:
Done. I also introduced a PR to make this change in the manifests of the plugin repo.